src/lightning_app/utilities/cli_helpers.py

```diff
@@ -119,3 +121,15 @@ def _retrieve_application_url_and_available_commands(app_id_or_name_or_url: Opti
                     raise Exception(f"The server didn't process the request properly. Found {resp.json()}")
                 return lightningapp.status.url, _extract_command_from_openapi(resp.json())
     return None, None
+
+
+def _arrow_time_callback(_ctx, _param, value, arw_now=arrow.utcnow()) -> arrow.Arrow:
```

Comment:
can we make sure that any new code added to `src/lightning_app` is mypy compliant? 
Add the files to the [mypy configuration](https://github.com/Lightning-AI/lightning/blob/master/pyproject.toml#L27) and fix all the missing type annotations.